### ISSUE TITLE
Problems identifying the linux distribution

### ISSUE THREAD

**Issue by shazaum** [intel/libvpl#144]
I'm using the Debian distribution version Bullseye...
I saw that the bootstrap script searches for the ID_LIKE variable, but it does not exist in os-release

look at that:
```
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

my suggestion is to also check if this variable exists

**Comment by mav-intel** [comment]
This PR https://github.com/intel/libvpl/pull/163 should address this

**Comment by mav-intel** [comment]
Fixed in 5be9788d7b92f9828e469f256780b81777ea49d0
